src/components/profile/picks.svelte

```diff
@@ -16,10 +16,13 @@
 		picks = (await axios.get('/api/picks')).data;
 	}
 
-	function getTypedPick(pick: Pick & { User: User; collab: Collab }): Pick & { User: User } {
+	function getTypedPick(
+		pick: Pick & { User: User | null; Character: AnimeCharacter; collab: Collab }
+	): Pick & { User: User; Character: AnimeCharacter } {
 		return {
 			...pick,
-			User: pick.User
```

Comment:
There's a non null assertion here, which we want to prevent.
A pick can't be without a user, at least not yet, so an assertion isn't needed and is a result of mistyped data.

Reply:
removed in commit d2e81b3bce561618f67c28c3a573bc090a0b6480